### PR TITLE
Some more translations into spanish

### PR DESCRIPTION
**Pull request by rsf92**
Some more translations, awaiting @Oberlus's comments.

### CODE REVIEW

**Comment by rsf92** [comment]
I've had a look at the errors on the pipeline, but I can't see how they're related to my changes...

File: default/stringtables/es.txt [hunk 1]
```diff
@@ -1167,11 +1167,45 @@ Escalado de coste de partes
 #*RULE_TEST_STRING_DESC
 #* <not translated 1970-01-01 01:00:00>
 
-#*RULE_STARLANES_EVERYWHERE
-#* <not translated 1970-01-01 01:00:00>
+RULE_STARLANES_EVERYWHERE
+Vías Estelares Completas
```
Comment (Oberlus) [inline default/stringtables/es.txt]: "Vías estelares completas" (only first letter is capital, same for other entries, see another comment below).

Great, you found a much better translation for starlanes: vías estelares.
Whenever you find mentions to "línea(s) estelare(s)", don't hesitate to replace "línea" by "vía".

File: default/stringtables/es.txt [hunk 2]
```diff
@@ -1167,11 +1167,45 @@ Escalado de coste de partes
...
+Se crean vías estelares entre cada par de sistemas, independientemente de la geometría. Estas vías pueden, aún así, ser eliminada durante la partida. Estas vías no aparecen en el mapa de la galaxia.
+
+RULE_ALL_OBJECTS_VISIBLE
+Todos los objetos visibles
+
+RULE_ALL_OBJECTS_VISIBLE_DESC
+Todo sistema, planeta, edificio, flota, nave o campo puede ser detectado por todos los imperios con al menos visibilidad parcial.
+
+RULE_ALL_SYSTEMS_VISIBLE
+Todos los sismteas visibles
```
Comment (Oberlus) [inline default/stringtables/es.txt]: typo: sistemas

File: default/stringtables/es.txt [hunk 3]
```diff
@@ -1167,11 +1167,45 @@ Escalado de coste de partes
...
+
+RULE_ALL_SYSTEMS_VISIBLE_DESC
+La localización de todos los sistemas es conocida por todos los imperios.
+
+RULE_EXTRASOLAR_SHIP_DETECTION
+Detección de naves extrasolares
+
+RULE_EXTRASOLAR_SHIP_DETECTION_DESC
+Las naves fuera del sistema pueden otorgan capacidad de detección
```
Comment (Oberlus) [inline default/stringtables/es.txt]: I like this more:
"Las naves fuera de sistemas solares tienen visión"
It is more explicative for casual players than the English "provide detection" (provide who with detection? the ship provides itself with detection).
If you prefer to be stick to a more direct translation, it should be this:
"Las naves fuera de sistemas solares otorgan detección" (_sistemas_ in plura)

File: default/stringtables/es.txt [hunk 4]
```diff
@@ -1167,11 +1167,45 @@ Escalado de coste de partes
...
+
+RULE_UNSEEN_STEALTHY_PLANETS_INVISIBLE
+Planetas Sigilosos Indetectables
```
Comment (Oberlus) [inline default/stringtables/es.txt]: "Planetas sigilosos invisibles" (not indetectable!)

"Sigiloso" sounds so weird for planets, but using "ocultos" here could be confusing if we use "sigilo" and "sigiloso" everywhere for "stealth" and "stealthy".

Side note that applies to several entries: In Spanish, title-ish stuff is expected to capitalize the first letter of only the first word, "El señor de los anillos", "Resultados y discusión".

File: default/stringtables/es.txt [hunk 5]
```diff
@@ -1167,11 +1167,45 @@ Escalado de coste de partes
...
+
+RULE_UNSEEN_STEALTHY_PLANETS_INVISIBLE_DESC
+'''<u>Activado</u>: Los planetas con mayor sigilo que la capacidad de detección de un imperio no pueden ser detectadas simplemente con que dicho imperio disponga de un objeto en el mismo sistema,salvo que ese planeta ya haya sido previamente detectado por el imperio.
```
Comment (Oberlus) [inline default/stringtables/es.txt]: "sistema,salvo" -> "sistema, salvo" (lacks a space)

**Review by Oberlus — CHANGES_REQUESTED** [review]

**Comment by Oberlus** [comment]
> I've had a look at the errors on the pipeline, but I can't see how they're related to my changes...

No idea about that.
@geoffthemedio

**Comment by geoffthemedio** [comment]
> > I've had a look at the errors on the pipeline, but I can't see how they're related to my changes...
> 
> No idea about that. @geoffthemedio

Don't ask me about the unit tests or why `SmokeTestHostless` seemingly [randomly fails](https://github.com/freeorion/freeorion/runs/4856593083?check_suite_focus=true#step:13:5297)... I didn't set them up...

**Comment by Oberlus** [comment]
Good to merge

**Review by Oberlus — APPROVED** [review]

**Pull request merged**